packages/cli/generation/ir-migrations/src/ir-versions/IrVersions.ts

```diff
@@ -59,3 +59,4 @@ export { FernIrV57 as V57 } from "@fern-fern/ir-v57-sdk";
 export { FernIrV58 as V58 } from "@fern-fern/ir-v58-sdk";
 export { FernIrV59 as V59 } from "@fern-fern/ir-v59-sdk";
 export { FernIrV60 as V60 } from "@fern-fern/ir-v60-sdk";
+export { FernIr as V61 } from "@fern-fern/ir-v61-sdk";
```

Comment:
for consistency the API should be renamed to `FernIrV61` when checkpointing the old IR definition